cli/lib/commands/InitCommand.dart

```diff
@@ -34,36 +35,40 @@ class InitCommand extends Command {
 
     final srcDirectory = await Directory(config.sourceDirectory).create();
 
-    final exampleFile = await File(path.join(srcDirectory.absolute.path, "index.md")).create();
+    final exampleFile =
+        await File(path.join(srcDirectory.absolute.path, "index.md")).create();
     await exampleFile.writeAsString(_getConfigMdContent(config));
 
     final baseDirectory = await Directory(config.baseDirectory).create();
 
-		final baseArg = argResults!["base"] as String;
+    final baseArg = argResults!["base"] as String;
 
-		final repoName = baseArg.split("/").sublist(0, 2).join("/");
-		final repoPath = baseArg.split("/").sublist(2).join("/");
-		final repoApiContentsPath = "https://api.github.com/repos/$repoName/contents/$repoPath";
-		final httpResponse = await http.get(Uri.parse(repoApiContentsPath));
+    final repoName = baseArg.split("/").sublist(0, 2).join("/");
+    final repoPath = baseArg.split("/").sublist(2).join("/");
```

Comment:
here is some magic. possible to siplify things? Or at least comment how it works?